### PR TITLE
Feature/update incidents page to reports page

### PR DESCRIPTION
**Pull request by robert-petersen**
# Description

The final tweak to the renaming of "incidents" to "incident reports" in order to more correctly depict our collected data. The navbar and stats components have been updated to show the "incidents" page as a "reports" page now. The route was updated and changed in all of its instances from the "incidents" route to the "incident-reports" route. Files in the repo still say incidents including the incidents folder and all the components inside are called by "incident-----".  I'm not sure if we want to change this so the files are more clear for the next teams, which would probably be a good idea, but I wanted to discuss it with the team first before we change so many files.

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)
- [?] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] `npm test`

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by stevenpmcfarlane — APPROVED** [review]
Yeah, not sure what to do about the naming in the data itself. Another option might be to just provide documentation that says all instances of "incident-reports" correspond to "incidents" or something like that

**Review by rafmolina — APPROVED** [review]

**Comment by robert-petersen** [comment]
> Yeah, not sure what to do about the naming in the data itself. Another option might be to just provide documentation that says all instances of "incident-reports" correspond to "incidents" or something like that

I was thinking at the very least I'm going to change the folder all the incidents components are in into "incident-reports" so it's a little more clear.

File: src/components/NavBar/NavBar.js [hunk 1]
```diff
@@ -29,8 +29,12 @@ const NavBar = () => {
         <NavLink onClick={handleClick} className="nav-link" exact to="/">
           Home
         </NavLink>
-        <NavLink onClick={handleClick} className="nav-link" to="/incidents">
-          Incidents
+        <NavLink
+          onClick={handleClick}
+          className="nav-link"
+          to="/incident-reports"
```
Comment (Brandonx1123) [inline src/components/NavBar/NavBar.js]: I do like the new pathing, makes it more clear but however i do we might need to add documentation

File: src/components/Stats/Stats.js [hunk 2]
```diff
@@ -64,7 +64,7 @@ const Stats = () => {
           type="primary"
           shape="round"
         >
-          <Link to="/incidents">Incidents</Link>
+          <Link to="/incident-reports">Reports</Link>
```
Comment (Brandonx1123) [inline src/components/Stats/Stats.js:67]: I think we should make the link itself say "incident-reports" not just "reports"

**Pull request merged**